### DISCUSSION TITLE
Why this code does not print anything?

### DISCUSSION THREAD

**Discussion by sbacelar** [fonsp/Pluto.jl#1042] [Questions about using Pluto]
My code was originally like `println(n)`. I know that I can't use `print` in `Pluto` (the result goes to the console. But why this code does not show anything?
```
begin
for n ∈ 1:6
	n
end
end
```

**Comment by icweaver** [comment]
You can embed a little terminal directly into the Pluto notebook with `PlutoUI.jl`. Could you give this a try?

```julia
using PlutoUI: with_terminal

with_terminal() do
    for n ∈ 1:6
        println(n)
    end
end
```

You can also replace `println` with logging commands like `@info` if you like as well. For direct printing in the notebook, there is also:

```julia
using PlutoUI: Text

Text() do io
    for n ∈ 1:6
        println(io, n)
    end
end
```

**Comment by ctrekker** [comment]
There is also a [pull request (#437)](https://github.com/fonsp/Pluto.jl/pull/437) that will handle logging done through `@info`, `@warn`, etc. and display a timeline on the right side of the cell

**Reply by icweaver** [reply]
Oh yea!
